squid/templates/deployment.yaml

```diff
@@ -88,8 +89,9 @@ spec:
               mountPath: /var/spool/squid
         {{- if .Values.squidExporter.enabled }}
         - name: squid-exporter
-          image: "{{ .Values.squidExporter.image.repository }}:{{ .Values.squidExporter.image.tag }}"
-          imagePullPolicy: {{ .Values.squidExporter.image.pullPolicy }}
+          image: "{{ include "squid.image" . }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/usr/sbin/container-entrypoint.sh", "exporter"]
```

Comment:
We should be able to remove this as well. `exporter` is already passed as an arg to the default entrypoint.